import_test.go

```diff
@@ -274,6 +276,16 @@ func createContent(size int64, seed int64) ([]byte, digest.Digest) {
 	if err != nil {
 		panic(err)
 	}
+	wb := bytes.NewBuffer(nil)
+	cw, err := compression.CompressStream(wb, compression.Gzip)
+	if err != nil {
+		panic(err)
```

Comment:
can we use t.Fatal?

Reply:
Not necessary, these are errors which should never occur but I much prefer panic to assigning to `_`